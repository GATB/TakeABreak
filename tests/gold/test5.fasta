>inv_0 au
AGGATTTTTAAAATTAAATAATATAAATTGAAATTAGAATAAAATATAGTAAATTGAGGATTTTTAAAATTAAATAATATAAATTGAAATTAGAATAAAATATAGTAAATTGAGGATTTTTAAAATGTAAAAAAATAAAAAAATG
>vb
TTTCAATTTATATTATTTAATTTTAAAAATCCTCAATTTACTATATTTTATTCTAATTTCAATTTATATTATTTAATTTTAAAAATCCTCAATTTACTATATTTTATTCTAATTTCAATTTATATTTTCTTACTCTTTTATTGAA
>av'
AGGATTTTTAAAATTAAATAATATAAATTGAAATTAGAATAAAATATAGTAAATTGAGGATTTTTAAAATTAAATAATATAAATTGAAATTAGAATAAAATATAGTAAATTGAGGATTTTTAAAATTAAATAATATAAATTGAAA
>u'b
CATTTTTTTATTTTTTTACATTTTAAAAATCCTCAATTTACTATATTTTATTCTAATTTCAATTTATATTATTTAATTTTAAAAATCCTCAATTTACTATATTTTATTCTAATTTCAATTTATATTTTCTTACTCTTTTATTGAA
