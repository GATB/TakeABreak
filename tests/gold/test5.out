TakeABreak                              
    version                                  : 1.1.2
    gatb-core-library                        : 1.1.1
    supported_kmer_sizes                     : 32 64 96 128 256
Parameters                              
    Input data                              
        Reads                                    : /Users/clemaitr/DATA/NGSdata/LbFV_raw_data/Varaldi_LbFV_TTCAGC_L001_R1_001.fastq
    Graph                                   
        kmer-size                                : 127
        nb_banks                                 : 1
        solidity_kind                            : one
        abundance_min (used)                     : 500 
        abundance_max                            : 2147483647
        nb_solid_kmers                           : 112227
        nb_branching_nodes                       : 155
    Inversions                              
        repeat                                   : 110
        max_sim                                  : 99
        LCT                                      : 100
Results                                 
    nb_occurrences                           : 64
    nb_distinct_solutions                    : 1
    time                                     : 0.0 s
    output_files                            
        graph_file                               : newtests/test5.h5
        solution_file                            : newtests/test5.fasta
